content/en/docs/services/snippet-service/features.md

```diff
@@ -0,0 +1,26 @@
+---
+title: 'Features'
+description: 'What features does the Snippet Service provide?'
+date: '2024-07-12'
+categories: [snippet-service, automation, configuration]
+keywords: [device-configuration, snippet-service]
+weight: 2
+---
+
+## Description
+
+This feature allows users to pre-configure devices by specifying desired configurations that are automatically applied when a ResourceChanged event occurs. The newly introduced `snippet service` is responsible for managing this process.
```

Comment:
For consistency we should probably enclose all ResourceChanged words in "``", so all occurences are visually the same

Reply:
fixed